Can't pass map to nav2_bringup_launch.py file

<!--
For general questions, please ask on ROS answers: https://answers.ros.org, make sure to include at least the `ros2` tag and the rosdistro version you are running, e.g. `ardent`.
For general design discussions, please post on discourse: https://discourse.ros.org/c/ng-ros
Not sure if this is the right repository? Open an issue on https://github.com/ros-planning/navigation2
For Bug report or feature requests, please fill out the relevant category below
-->

## Bug report

**Required Info:**

- Operating System:
  - Ubuntu 18.04
- Version or commit hash:
  - master branch since 2873bfc3468a25a47483f22d17a232a83355dfc6

- DDS implementation:
  - Fast-RTPS (default)

#### Steps to reproduce issue
<!-- Detailed instructions on how to reliably reproduce this issue http://sscce.org/
``` code that can be copy-pasted is preferred ``` -->
```
ros2 launch nav2_bringup nav2_bringup_launch.py map:=$PWD/src/navigation2/nav2_system_tests/maps/map_circular.yaml autostart:=True
```

#### Expected behavior
Nodes should start correctly with map given.

#### Actual behavior
```
[map_server-1] [INFO] [map_server]: Configuring
[map_server-1] [ERROR] []: Caught exception in callback for transition 10
[map_server-1] [ERROR] []: Original error: Could not open 'map.yaml': file not found
[map_server-1] [WARN] []: Error occurred while doing error handling.
[map_server-1] [FATAL] [map_server]: Lifecycle node entered error state
[lifecycle_manager-8] [ERROR] [lifecycle_manager]: Failed to change state for node: map_server
[lifecycle_manager-8] [ERROR] [lifecycle_manager]: Failed to bring up nodes: aborting bringup
````
#### Additional information
I bisected it to the commit above, trying to debug 'why' this is the case now.

--- a/nav2_bringup/bringup/launch/nav2_bringup_launch.py
+++ b/nav2_bringup/bringup/launch/nav2_bringup_launch.py
@@ -47,7 +47,7 @@ def generate_launch_description():
 
     declare_namespace_cmd = DeclareLaunchArgument(
         'namespace',
-        default_value='navigation',
+        default_value='',
         description='Top-level namespace')
 
     declare_use_namespace_cmd = DeclareLaunchArgument(

